tests/v2/validation/airgap/airgap.go

```diff
@@ -0,0 +1,33 @@
+package airgap
```

Comment:
can you add a README.md file in this folder?

Reply:
+1, would be helpful so we have a reference 😄

Reply:
Got it up